store/dataStore.test.ts

```diff
@@ -32,4 +38,51 @@ describe('Data Store', () => {
     expect(result.current.user.id).toBe('');
     expect(result.current.user.email).toBe('');
   });
+  it('Check the default NFL Teams state', () => {
+    const { result } = renderHook(() => useDataStore());
+    expect(result.current.NFLTeams.teamName).toBe('');
+    expect(result.current.NFLTeams.teamLogo).toBe('');
+  });
+  it('Check the updated NFL Teams state', () => {
+    const { result } = renderHook(() => useDataStore());
+
+    act(() => {
+      result.current.updateNFLTeams(teamName, teamLogo);
+    });
+
+    expect(result.current.NFLTeams.teamName).toBe(teamName);
+    expect(result.current.NFLTeams.teamLogo).toBe(teamLogo);
+  });
+  it('Check the default userWeeklyPick state', () => {
+    const { result } = renderHook(() => useDataStore());
+    expect(result.current.user.id).toBe('');
+    expect(result.current.userWeeklyPick.weekNumber).toBe('');
+  });
+  it('Check the updated userWeeklyPick state', () => {
+    const { result } = renderHook(() => useDataStore());
+
+    act(() => {
+      result.current.updateUserWeeklyPick(userId, weekNumber);
+    });
+
+    expect(result.current.user.id).toBe(userId);
+    expect(result.current.userWeeklyPick.weekNumber).toBe(weekNumber);
+  });
+  it('Check the default updateWeeklyPicks state', () => {
+    const { result } = renderHook(() => useDataStore());
+    expect(result.current.weeklyPicks.gameId).toBe('');
+    expect(result.current.weeklyPicks.gameWeekId).toBe('');
+    expect(result.current.weeklyPicks.userResults).toBe('');
+  });
+  it('Check the updated updateWeeklyPicks state', () => {
+    const { result } = renderHook(() => useDataStore());
+
+    act(() => {
+      result.current.updateWeeklyPicks(gameId, gameWeekId, userResults);
+    });
+
+    expect(result.current.weeklyPicks.gameId).toBe(gameId);
+    expect(result.current.weeklyPicks.gameWeekId).toBe(gameWeekId);
+    expect(result.current.weeklyPicks.userResults).toBe(userResults);
+  });
```

Comment:
Do you think we need to add reset functions for these?  If so, add them to the `dataStore.tsx` file and then update the test.

Reply:
When you state reset, do you mean clear out the choice after they have selected? @chris-nowicki

Reply:
Per design, we do not.

Reply:
cool.  thanks!